### ISSUE TITLE
samtools view does not initiate on Mac OS

### ISSUE THREAD

**Issue by PaulChed** [samtools/samtools#1709]
Error utilising samtools view
$ samtools view
dyld[3478]: Library not loaded: '@rpath/libcrypto.1.0.0.dylib'
  Referenced from: '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/samtools'
  Reason: tried: '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/usr/local/lib/libcrypto.1.0.0.dylib' (no such file), '/usr/lib/libcrypto.1.0.0.dylib' (no such file)
Abort trap: 6

#### Are you using the latest version of [samtools](https://github.com/samtools/samtools/releases/latest) and [HTSlib](https://github.com/samtools/htslib/releases/latest)? If not, please specify.
(run `samtools --version`) - this also produces an error message:
$ samtools --version
dyld[3674]: Library not loaded: '@rpath/libcrypto.1.0.0.dylib'
  Referenced from: '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/samtools'
  Reason: tried: '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/Users/paulcheddie/miniconda3/envs/bioinformatics_venv/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/usr/local/lib/libcrypto.1.0.0.dylib' (no such file), '/usr/lib/libcrypto.1.0.0.dylib' (no such file)
Abort trap: 6

 #### Please describe your environment.
- OS (run `uname -sr` on Linux/Mac OS or ` wmic os get Caption, Version` on Windows)
- **Darwin 21.6.0**
- machine architecture (run `uname -m` on Linux/Mac OS or ` wmic os get OSArchitecture` on Windows)
- **x86_64**
- compiler (run `gcc --version` or `clang --version`)
- **clang version 13.1.6**

**Comment by jmarshall** [comment]
The issue is with the way you are using conda, so this is really a condo-related issue.

The problem is that you have not activated the **conda-forge** channel as described in the [bioconda documentation](https://bioconda.github.io/#usage). Recent bioconda samtools packages use dependencies from conda-forge, so not having it means conda will choose an ancient samtools package that does not track its libcrypto dependency correctly — with the results you have seen.

In summary, you need to set up your conda channels as described in the [bioconda setup documentation](https://bioconda.github.io/#usage). Duplicate of #1686.

**Comment by PaulChed** [comment]
> The issue is with the way you are using conda, so this is really a conda-related issue.
> 
> The problem is that you have not activated the **conda-forge** channel as described in the [bioconda documentation](https://bioconda.github.io/#usage). Recent bioconda samtools packages use dependencies from conda-forge, so not having it means conda will choose an ancient samtools package that does not track its libcrypto dependency correctly — with the results you have seen.
> 
> In summary, you need to set up your conda channels as described in the [bioconda setup documentation](https://bioconda.github.io/#usage). Duplicate of #1686.

So

**Comment by PaulChed** [comment]
> The issue is with the way you are using conda, so this is really a conda-related issue.
> 
> The problem is that you have not activated the **conda-forge** channel as described in the [bioconda documentation](https://bioconda.github.io/#usage). Recent bioconda samtools packages use dependencies from conda-forge, so not having it means conda will choose an ancient samtools package that does not track its libcrypto dependency correctly — with the results you have seen.
> 
> In summary, you need to set up your conda channels as described in the [bioconda setup documentation](https://bioconda.github.io/#usage). Duplicate of #1686.

So I reconfigured the channels but ended up with the error. Nothing seems to be working. 

Error:
% samtools view
dyld[10307]: Library not loaded: '@rpath/libcrypto.1.0.0.dylib'
  Referenced from: '/Users/paulcheddie/miniconda3/bin/samtools'
  Reason: tried: '/Users/paulcheddie/miniconda3/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/Users/paulcheddie/miniconda3/bin/../lib/libcrypto.1.0.0.dylib' (no such file), '/usr/local/lib/libcrypto.1.0.0.dylib' (no such file), '/usr/lib/libcrypto.1.0.0.dylib' (no such file)
zsh: abort      samtools view

**Comment by jkbonfield** [comment]
My experience of Conda is when it goes wrong, create a new environment and try again from there. (It can become dependency hell otherwise.)  You may also get more mileage from using mamba instead of conda, and/or explicitly specifying the version of samtools you wish to install.

However as John says, these are Conda usage problems and not a samtools one.  If you think there are (bio)conda bugs, which there most definitely have been, please report these using their own github pages.  If it's just conda usage questions, then general bioinformatics web sites such as BioStars may reach a more appropriate audience.
